Sources/NestKit/NestInfoController.swift

```diff
@@ -19,6 +19,24 @@ public struct NestInfoController {
         return nestInfo
     }
 
+    /// Get NestInfo.Command from `owner/repository`
+    public func fetchCommand(reference: String, version: String) -> NestInfo.Command? {
+         return getInfo().commands
+            .first {
+                let command = $0.value
+                    .first {
+                        switch $0.manufacturer {
+                        case let .artifactBundle(sourceInfo):
+                            return sourceInfo.repository?.reference.referenceName == reference
+                        case let .localBuild(repository):
+                            return repository.reference.referenceName == reference
+                        }
+                    }
+                return command != nil
```

Comment:
This may be  straightforward.
```suggestion
                $0.value
                    .contains {
                        switch $0.manufacturer {
                        case let .artifactBundle(sourceInfo):
                            return sourceInfo.repository?.reference.referenceName == reference
                        case let .localBuild(repository):
                            return repository.reference.referenceName == reference
                        }
                    }
```

Reply:
Thanks!
I've made the corrections here. (https://github.com/mtj0928/nest/pull/36/commits/3b4f4650df3ac111992cd37b34ba792fa6605316)